locales/en.default.schema.json

```diff
@@ -1978,6 +1978,162 @@
       "presets": {
         "name": "Video"
       }
+    },
+    "slideshow": {
+      "name": "Slideshow",
+      "settings": {
+        "layout": {
+          "label": "Layout",
+          "options__1": {
+            "label": "Full width"
+          },
+          "options__2": {
+            "label": "Grid"
+          },
+          "options__3": {
+            "label": "Grid and peek"
```

Comment:
```suggestion
            "label": "Grid with a peek"
```

Reply:
👍  if I do this layout in a follow up PR I might remove it from the settings for now 🤔